tool/grind.dart

```diff
@@ -84,50 +83,31 @@ serveCustomBackend() {
   run('sed', arguments: [
     '-i',
     's,https://dart-services.appspot.com,${Platform.environment[backendVariable]},g',
-    'build/web/scripts/main.dart.js',
-    'build/web/scripts/embed.dart.js'
+    'build/scripts/main.dart.js',
+    'build/scripts/embed.dart.js'
   ]);
-  run('pub', arguments: ['run', 'dhttpd', '-p', '8000', '--path=build/web']);
+  run('pub', arguments: ['run', 'dhttpd', '-p', '8000', '--path=build']);
```

Comment:
This no longer builds into `web/`?

Reply:
Looks like just into `build/`. How does webdev / build_runner choose which entrypoints to compile?

Reply:
@devoncarew As discussed offline, this is a complex question.  I think a build.yaml can control that, but we are using the defaults which seem to do the right thing AFAICT (building the main and embedded entry points).